### ISSUE TITLE
Return the histogram of the training

### ISSUE THREAD

**Issue by Maxime-POULAIN-Verlingue** [MAIF/melusine#114]
Hello,

**Description of Problem**: We wanted to get the values of metrics (like accuracy) from the train function. It would give us the possibility to plot the evolution of the accuracy during the different epochs of the training, or get the last value as the accuracy on the training dataset.

**Overview of the Solution**: To do that, we just need to store the output of the fit function in the train function of melusine. Then we return the value of this output.

![image](https://user-images.githubusercontent.com/94903022/143227327-58831ca5-e03c-4a01-8801-1ec7241866d9.png)

**Possibility**:
![image](https://user-images.githubusercontent.com/94903022/143228197-c5be3962-79f4-4a40-9218-1ea3df08babe.png)

**Comment by TFA-MAIF** [comment]
Hi Maxime, 

This issue is interesting and represent low risks to integrate. 
Would you be willing to submit a PR (and link it to this issue)?

Best regards,
